ekundayo-ab/hello-books
# #153016280 Add seed data for categories

### What does this PR do?
Ensures seed data are made available to pre-populate the categories table

### Description of Task to be completed?
Manually create data which can be seeded into the categories table using `sequelize-cli`

### How should this be manually tested?
1. Pull in latest changes from remote repository to local repository
2. Fire up terminal run `npm install`
3. To seed data do `npm run db:reset`
4. execute `npm run start:dev` and then `npm run build:dev` to start up application
5. Visit url supplied by the application usually **http://localhost:3000**
6. login with `ekprogs@gmail.com` and `dayo` for admin access, while for normal user access simply register a new user on the application
7. After logging in to the application, Visit the **Shelf** and **Admin** page to see seeded categories

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
#153016280 Add seed data for categories

### Screenshots (if appropriate)
N/A

### Questions:
N/A
